data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
+Burchel Beam Repeater,diableavionics_burchel,1,0.5,600,600,300,,,,75,8,12,1,3,ENERGY,,240,,,0.48,0.02,,,,,1000000,,,,,,,"PD,ANTI_FTR,PD_ALSO","pd9,beam8, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
+Glaux Lance Repeater,diableavionics_glaux,2,0.5,2250,700,687,,,,16,11,8,0.727,2,ENERGY,,618,,,0.48,0.02,,,,,1000000,,,,,,,,"energy14, beam10, diable_weapon_bp, rare_bp",,Diable Avionics,,Strike,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Hexafire Scatter-Cannon,diableavionics_hexafire,1,,1750,700,,100,30,10,15,11,,,,ENERGY,110,,0,2.5,,,2,16,0.6,6,,600,,,100,,,,"energy13, diable_weapon_bp, rare_bp",,Diable Avionics,,General,,,,,%s: projectiles deal up to %s extra damage the higher the firing ship's %s is.,Flux charged ordinance|50%| flux level,,,,
-Twin-Hexafire Scatter-Cannon,diableavionics_hexafire2,2,,5000,700,,100,30,10,8,25,,,,ENERGY,110,,0,2.5,,,2,16,0.3,6,,600,,,120,,,,"energy17, diable_weapon_bp, rare_bp",,Diable Avionics,,General,,,,,%s: projectiles deal up to %s extra damage the higher the firing ship's %s is.,Flux charged ordinance|50%|flux level,,,,
+Hexafire Scatter-Cannon,diableavionics_hexafire,1,,1750,700,,120,30,10,15,11,,,,ENERGY,120,,0,2.5,,,2,16,0.6,6,,1200,,,100,,,,"energy13, diable_weapon_bp, rare_bp",,Diable Avionics,,General,,,,,%s: projectiles deal up to %s extra damage the higher the firing ship's %s is.,Flux charged ordinance|100%| flux level,,,,
+Twin-Hexafire Scatter-Cannon,diableavionics_hexafire2,2,,5000,700,,120,30,10,8,25,,,,ENERGY,120,,0,2.5,,,2,16,0.3,6,,1200,,,120,,,,"energy17, diable_weapon_bp, rare_bp",,Diable Avionics,,General,,,,,%s: projectiles deal up to %s extra damage the higher the firing ship's %s is.,Flux charged ordinance|100%|flux level,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Lazyhorn Warp Projector,diableavionics_lazyhorn,1,,1250,600,,200,,40,25,12,,,,FRAGMENTATION,100,,0,3.4,7,0.1,45,45,0,0,,100,600,3,100,-2,5,"PD,ANTI_FTR","pd14, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense (Area),,,Fast,Terrible,,,,,,
-State Support Beam,diableavionics_state,1,,1500,800,400,,,0,30,11,,,,ENERGY,,393,0.1,0.1,2,1.925,,,,,3000,,,,,,,NO_TURN_RATE_BOOST_WHEN_IDLE,"beam12, energy11, diable_weapon_bp",,Diable Avionics,,General,,,,,,,,,,
-SRAB Heavy Emitter,diableavionics_srab,3,0.75,2300,800,,600,400,50,7,13,,,,ENERGY,750,,0,3,,,,,,,,400,100,2.8,200,,5,,"energy15, LR, diable_weapon_bp, rare_bp",,Diable Avionics,,Long-range pressure,,Poor,Fast,,%s: The projectiles have limited target tracking capacity.,Target Tracking,%s: The projectiles can be intercepted by regular PD weapons.,Unstable Projectile,,
+State Support Beam,diableavionics_state,1,,1500,800,400,,,0,30,12,,,,HIGH_EXPLOSIVE,,320,0.1,0.1,2,1.925,,,,,3000,,,,,,,NO_TURN_RATE_BOOST_WHEN_IDLE,"beam12, energy11, diable_weapon_bp",,Diable Avionics,,General,,,,,,,,,,
```

Comment:
State Support Beam OP cost increased to 12 (from 11). Now deals HE damage. Flux per second reduced to 320 (from 393).